features/mock_for_button_id.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Mock for button id
+  As a user
+  I want to open page with two buttons
+  In order to check payment process for two buttons
+
+  Background:
+    Given JavaScript configuration is set for scenario based on scenario's @config tag
+    And User opens prepared payment form page WITH_ADDITIONAL_BUTTON
+
+  @base_config
+  Scenario: Click on button configured as button id
+    When User fills payment form with defined card VISA_CARD
+    And THREEDQUERY mock response is set to "ENROLLED_Y"
+    And User clicks Pay button - AUTH response is set to "OK"
```

Comment:
You can also add this step: Then User will see payment status information: "Payment has been successfully processed" to be sure that payment has been processed correctly

Reply:
dome